### ISSUE TITLE
values for gas consumption 4x too high

### ISSUE THREAD

**Issue by markusschloesser** [bosch-thermostat/home-assistant-bosch-custom-component#229]
**Describe the bug**
Hi, just updated from 0.17 to 0.19.4 and added gas consumption to the energy board. But it is roughly 4 times too high.
I moved the statistics according to the description and see values going back 1 month. 
I didn't need to enable the 2 ractual values (dwh and ch), probably because I had them already enabled before. Nevertheless the "external" is shown. 
Values reported in Energy Board for October: 4.722kwh 
Values in m3 from Gas meter: 117m3 (should be the equivalent of roughly 1,170kwh (?)

**Screenshots**
<img width="738" alt="image" src="https://user-images.githubusercontent.com/59286549/200197004-f19a90bb-6a3f-41b8-9f5a-3e8a457415cf.png">

**Version**
Home Assistant 2022.11.1
Supervisor 2022.10.2
Operating System 9.3
Frontend 20221102.1 - latest
0.19.4

** Debug SCAN **
https://drive.google.com/file/d/1wciRrO59Lwz8EDqeNe_Ns3cV4YiUNkim/view?usp=share_link

**Additional context**
Firmware: 01.10.03 IVT

EDIT (further info):
Been using the integration successfully for months and am very grateful for it! :-)
AFAIK everything else is working fine after the update, except those values. On the other hand, I don't know how high those values were before.

Looking at the json, one thing that strikes me as odd is /heatingCircuits/hc1/roomtemperature with a float value of `value": 3200.0,` but that probably isn't the problem ;-)

**Comment by pszafer** [comment]
If you take `actualChPower` from scan:
```
"recording": [
        {
          "y": 598,
          "c": 59
        },
        {
          "y": 0,
          "c": 60
        },
        {
          "y": 0,
          "c": 60
        },
        {
          "y": 0,
          "c": 60
        },
        {
          "y": 0,
          "c": 60
        },
        {
          "y": 0,
          "c": 59
        },
        {
          "y": 39,
          "c": 60
        },
        {
          "y": 287,
          "c": 60
        },
        {
          "y": 304,
          "c": 60
        },
        {
          "y": 1174,
          "c": 60
        },
        {
          "y": 1375,
          "c": 59
        },
        {
          "y": 1375,
          "c": 60
        },
        {
          "y": 900,
          "c": 60
        },
        {
          "y": 200,
          "c": 60
        },
        {
          "y": 400,
          "c": 60
        },
        {
          "y": 1079,
          "c": 59
        },
        {
          "y": 1375,
          "c": 60
        },
        {
          "y": 849,
          "c": 60
        },
        {
          "y": 1129,
          "c": 60
        },
        {
          "y": 925,
          "c": 60
        },
        {
          "y": 1287,
          "c": 60
        },
        {
          "y": 1147,
          "c": 59
        },
        {
          "y": 1141,
          "c": 60
        },
        {
          "y": 800,
          "c": 60
        }
      ]
      ```

do sum all (y/c) and you got around 294kWh.
First of all energy used to heat by heater is not equal to energy used in gas.
Second, there is no sensor AFAIK for this energy usage and it's estimated value.

3200 means that sensor is not connected or corrupted.

**Comment by markusschloesser** [comment]
Thank you for your reply, appreciate it!

The thing is, I only enabled dwh and ch because it says so on https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/wiki/New-API-support. 
I am fine with just one value and will monitor it in the next couple of weeks.

2 Questions:
1. Would it be OK for you, if I edit the wiki pages? For example the dwh/ch problem, and also it would imho be helpful information if on https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/wiki/Move-from-0.17-to-0.18-with-new-statistics it says between 1) and 2) to update the integration first (you might laugh, but I did the backup, and then searched for `move_old_statistic_data` before updating ;-) )
2. Should the (too high/old) values vanish or be replaced in the energy dashboard? Because it still shows the old /too high values. And how would I do this?
